build/ublue-os-just/10-update.just

```diff
@@ -104,15 +106,19 @@ toggle-updates ACTION="prompt":
       exit 0
     fi
     if [ "${OPTION,,}" == "enable" ]; then
-      sudo systemctl enable ublue-update.timer
-      if systemctl is-enabled flatpak-system-update.timer | grep -q disabled; then
+      if systemctl is-enabled ublue-update.timer 2> /dev/null | grep -q not-found; then
         sudo systemctl enable --now flatpak-system-update.timer rpm-ostreed-automatic.timer
         systemctl enable --now --user flatpak-user-update.timer
+      else
+        sudo systemctl enable ublue-update.timer
```

Comment:
Ublue update is not required to be installed 

Its only on Bazzite/bluefin/Aurora.